src/Server.php

```diff
@@ -75,6 +75,12 @@ class Server extends EventEmitter
      * $http = new React\Http\Server($socket);
      * ```
      *
+     * When HTTP/1.1 clients want to send a bigger request body, they MAY send only the request
+     * headers with an additional `Expect: 100-continue` header and wait before
+     * sending the actual (large) message body.
+     * In this case the server will automatically send an intermediary `HTTP/1.1 100 Continue` response to the client.
+     * This ensures you will receive the request body without a delay as expected.
```

Comment:
Does this really make sense as documentation for the ctor? :roll_eyes: 

Also, may I ask you to wrap the lines at a reasonable length? Thanks!